test/fixtures/movies.yml

```diff
@@ -0,0 +1,13 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```

Comment:
should we have this test folder with files ? or maybe Spec files will be enough for tests?

Reply:
That folder is crated by rails by default if you initialize the project with `Test::Unit`. Can be deleted cause we plan to use Rspec.